src/service/navigation.js

```diff
@@ -173,6 +180,19 @@ export class Navigation {
     this.navigateToMutators_ = new PriorityQueue();
   }
 
+  /**
+   *  Sets values for boolean fields isTrustedViewer_ and isLocalViewer_
+   */
+  setTrustedViewerStatus() {
```

Comment:
This should be a private function, but let's remove and just inline it in the constructor.

Reply:
Done.